.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,369 @@
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/cache@v1
+              with:
+                  path: ./node_modules
+                  key: ${{github.sha}}-node_modules
+            - uses: actions/cache@v1
+              with:
+                  path: ./dist/i18n
+                  key: i18n
+            - uses: actions/cache@v1
+              with:
+                  path: ./dist/components
+                  key: ${{ github.sha }}-components
+            - run: npm run build:components
+
+    build-doc-lib:
+        runs-on: ubuntu-latest
+        name: Build /doc-lib
+        needs: [lint-projects]
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/cache@v1
+              with:
+                  path: ./node_modules
+                  key: ${{github.sha}}-node_modules
+            - uses: actions/cache@v1
+              with:
+                  path: ./dist/doc-lib
+                  key: doc-lib
+            - run: npm run build:doc-lib
+
+    build-app:
+        runs-on: ubuntu-latest
+        name: Build /examples-app
+        needs: [build-components-lib, build-doc-lib]
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/cache@v1
+              with:
+                  path: ./node_modules
+                  key: ${{github.sha}}-node_modules
+            - uses: actions/cache@v1
+              with:
+                  path: ./dist/components
+                  key: ${{ github.sha }}-components
+            - uses: actions/cache@v1
+              with:
+                  path: ./dist/doc-lib
+                  key: doc-lib
+            - uses: actions/cache@v1
+              with:
+                  path: ./dist/i18n
+                  key: i18n
+            - uses: actions/cache@v1
+              with:
+                  path: ./dist/examples
+                  key: examples
+            - run: |
+                  npm run build-components-doc
+                  npm run build-examples-doc
+                  npm run generate-translations
+                  npm run generate-example-translations
+                  npm run build:examples-prod
+
+    unit-testing:
+        runs-on: ubuntu-latest
+        name: Unit tests
+        needs: [build-i18n]
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/cache@v1
+              with:
+                  path: ./node_modules
+                  key: ${{github.sha}}-node_modules
+            - uses: actions/cache@v1
+              with:
+                  path: ./dist/i18n
+                  key: i18n
+            - uses: actions/cache@v1
+              with:
+                  path: ./coverage
+                  key: coverage
+            - name: Run unit-tests
+              run: |
+                  npm run test:ci:components
+                  npm run test:ci:doc-lib
+                  npm run test:ci:i18n
+            - name: Upload coverage to Codecov
```

Comment:
Does this upload the Codecov report, and if so, why do we need to keep the .travis.yml? And if we are keeping it and requiring the build to run to post coverage results, doesn't that reduce the benefit of switching over to GH Actions?